onset_maproom/calc.py

```diff
@@ -209,34 +203,33 @@ def cess_date(
     min_dry_days, 
     time_coord="T"
 ):
-    """ Computes single cessation date from daily soil moisture data.
-
-        Cessation Date finds the first day of the first dry spell where soil moisture falls 
-        below the defined dry threshold value for a minimum number of days.
-
-        Parameters
-        ----------
-        soil_moisture : DataArray
-                    Array of daily soil moisture.
-        dry_thresh : int
-                    Soil moisture threshold to determine dry day.
-        min_dry_days : int
-                    Minimum number of dry days in a row to be considered a dry spell.
-        time_coord : str, optional
-                    Time grid in 'soil_moisture' (the default is "T", which implies 
-                    naming convention of time grid as "T" in 'soil_moisture')
-        Returns
-        -------
-        cess_delta : DataArray 
-                    Cessation dates as timedelta values.
-        See Also
-        --------
-        Notes
-        -----
-        Examples
-        --------
-
-"""
+    """Calculate cessation date.
+
+    Find first day of the first dry spell where: 
+        Soil moisture falls below `dry_thresh` for `min_dry_days` days.
+
+    Parameters
+    ----------
+    soil_moisture : DataArray
+        Array of daily soil moisture.
+    dry_thresh : int
+        Soil moisture threshold to determine dry day.
+    min_dry_days : int
```

Comment:
that is unfortunate naming as, compared to onset, it resembles the _spell_length Parameters, wheras min_dry_days, in onset, is something else... Can fix that in another PR.